.github/actions/security-controls/action.yml

```
@@ -0,0 +1,25 @@
+name: "Generate ITSG-33 control GitHub issues"
+description: "Create github issues for each controls provided in the csv file"
+
+inputs:
+  token:
+    description: "Token for allowing the action to create issues"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout Code
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      repository: cds-snc/security-tools
+      path: tools/itsg33-issue-generator
```

You just have to add a `with` block and this should do the trick:
```suggestion
      with:
        repository: cds-snc/security-tools
        path: tools/itsg33-issue-generator
```

---

Done!